sdk_container/src/third_party/coreos-overlay/coreos-base/common-oem-files/common-oem-files-0.ebuild

```
@@ -4,8 +4,9 @@
 EAPI=8
 
 OEMIDS=(
-    qemu
     azure
+    ec2
```

```suggestion
    ami
```
Here, too, we need to stick to the `ami` name because of existing installs.